packages/angular/src/generators/application/application.spec.ts

```diff
@@ -43,6 +43,19 @@ describe('app', () => {
       expect(readProjectConfiguration(appTree, 'my-app-e2e')).toMatchSnapshot();
     });
 
+    it('should not produce tests when UnitTestRunner = none', async () => {
+      // ACT
+      await generateApp(appTree, 'my-app', {
+        unitTestRunner: UnitTestRunner.None,
+      });
+
+      expect(readProjectConfiguration(appTree, 'my-app')).toMatchSnapshot();
+      expect(readProjectConfiguration(appTree, 'my-app-e2e')).toMatchSnapshot();
+      expect(
+        appTree.exists('apps/my-app/src/app/app.component.spec.ts')
+      ).toBeFalsy();
```

Comment:
nit: there are assertions here that don't provide much value for the test case, checking the spec file doesn't exist should be enough, maybe you could also check the `test` target is not generated in the project configuration:
```suggestion
      const { targets } = readProjectConfiguration(appTree, 'my-app');
      expect(targets.test).toBeFalsy();
      expect(
        appTree.exists('apps/my-app/src/app/app.component.spec.ts')
      ).toBeFalsy();
```

Reply:
yeah the e2e one is definitely not needed and the other snapshot was to check for the test target, so definitely could be simplified